privacyidea/lib/policy.py

```diff
@@ -1017,20 +1026,15 @@ def ui_get_enroll_tokentypes(self, client, logged_in_user):
         :type logged_in_user: dict
         :return: list of token types, the user may enroll
         """
-        from privacyidea.lib.auth import ROLE
         enroll_types = {}
-        role = logged_in_user.get("role")
-        if role == ROLE.ADMIN:
-            # If the logged-in user is an admin, we match for username/adminrealm
-            user_name = logged_in_user.get("username")
-            user_object = None
-            admin_realm = logged_in_user.get("realm")
-        else:
-            # If the logged-in user is a user, we pass an user object to allow matching for userinfo attributes
-            user_name = None
-            user_object = User(logged_in_user.get("username"),
-                               logged_in_user.get("realm"))
-            admin_realm = None
+        # In this case we do not distinguish the userobject as for whom an administrator would enroll a token
+        # We simply want to know, which tokentypes a user or an admin in generally allowed to enroll. This is
+        # why we pass an empty params.
+        (role, username, userrealm, adminuser, adminrealm) = determine_logged_in_userparams(logged_in_user, {})
```

Comment:
sometimes the returned `scope` is called "`scope"` and sometimes "`role`". We should use consistent naming here.

Reply:
Actually what is returned is the "role" of the user, which is used as "scope" in the policies.
So I will check to change this to role.

Reply:
done in 7649246d672eb64ff314a7ec69826f2ea43fc933